yoti-sdk/src/main/java/com/yoti/mobile/android/sdk/kernelSDK/KernelSDKIntentService.java

```diff
@@ -150,8 +151,15 @@ private void handleActionStartScenario(String useCaseId, ResultReceiver resultRe
             intent.setData(uri);
         }
 
+        int flags;
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
Do we need the version check here? FLAG_IMMUTABLE is a constant and will be ignored on older versions